client/components/platform-checkout/save-user/phone-number-input.js

```diff
@@ -19,7 +18,8 @@ const PhoneNumberInput = ( { handlePhoneNumberChange } ) => {
 	const [ inputInstance, setInputInstance ] = useState( null );
 	const [ isValid, setIsValid ] = useState( true );
 
-	const handlePhoneNumberInputChange = ( value = inputValue ) => {
+	const handlePhoneNumberInputChange = ( e ) => {
```

Comment:
I updated this, otherwise it shows something like `[Object] [object]` in the phone field.

Reply:
Thanks! I forgot to mention that was broken in the diff I sent you. 🙏